src/Cms/File.php

```diff
@@ -357,12 +357,14 @@ public function mediaUrl(): string
     }
 
     /**
-     * @deprecated 3.0.0 Use `File::content()` instead
+     * @deprecated 3.0.0 Use `$file->content()` instead
```

Comment:
Please don't change the notation here (and everywhere else), this will break the automatic linking in the Reference.

Reply:
Oh, I didn't know we had that for the `@deprecated` markers. Fixed now. 👍

Reply:
Just for reference where this "magic" happens: https://github.com/getkirby/getkirby.com/blob/76de1d2e9ba132e604bc8b47b3b82cb096190eaa/site/plugins/site/helpers.php#L184-L210